plugins/trace_api_plugin/trace_api_plugin.cpp

```diff
@@ -100,6 +100,9 @@ struct trace_api_common_impl {
                   "the location of the trace directory (absolute path or relative to application data dir)");
       cfg_options("trace-slice-stride", bpo::value<uint32_t>()->default_value(10'000),
                   "the number of blocks each \"slice\" of trace data will contain on the filesystem");
+      cfg_options("trace-minimum-irreversible-history-blocks", boost::program_options::value<int32_t>()->default_value(-1),
```

Comment:
need to remove `trace-minimum-irreversible-history-us` below however, I would argue that this parameter should not be part of the common config options in the future but for now it is fine.  If we separate into a reader-only plugin, that plugin would still have the "common" configs but this one would be irrelevant as it would be ignored.